domains/creepycat.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "For personal website",
+  "owner": {
+    "username": "creepycat",
```

```suggestion
    "username": "CreepyTheCat",
```
This must be your GitHub username.